submissions/evgeniy24/friends_app/app.js

```diff
@@ -0,0 +1,228 @@
+const REQUEST_URL = 'https://randomuser.me/api/?results=20',
+    FRIENDS_LIST_WRAP = document.querySelector('.friendsList-wrap'),
+    HEADER = document.querySelector('.header-name');
+
+let friendsList = [],
+    searchResultList,
+    sortedList,
+    filteredByGenderList,
+    scrollDepth;
+
+const sendRequest = function (url) {
+    return fetch(url)
+        .then(handleErrors)
+        .then(function (response) {
+            return response.json();
+        });
+}
+
+function handleErrors(response) {
+    if (response.status > 400) {
```

Comment:
```suggestion
    if (response.ok) {
```

Reply:
I  used   if ( !response.ok ), cause it is Errors response check
Fixed!